docs/data/material/guides/next-js-app-router/next-js-app-router.md

```diff
@@ -0,0 +1,227 @@
+# Next.js App Router
+
+<p class="description">Learn how to use MUI libraries with the Next.js App Router</p>
+
+## Next.js and React Server Components
+
+Next.js's App Router implements React Server Components, which is a [new feature](https://github.com/reactjs/rfcs/blob/main/text/0227-server-module-conventions.md#changes-since-v1) in React.
+
+To support the App Router, currently all components and hooks from MUI libraries (Material UI, Joy UI, Base UI etc) are designated as client components and exported with the "use client" directive.
+
+:::info
+Don't need a guide? Use one of these example repos and get started:
+
+- [Material UI](https://github.com/mui/material-ui/blob/master/examples/material-next-app-router-ts)
+- [Joy UI](https://github.com/mui/material-ui/blob/master/examples/joy-next-app-router-ts)
+- [Base UI with Tailwind CSS](https://github.com/mui/material-ui/blob/master/examples/base-next-app-router-tailwind-ts)
```

Comment:
Would be great to add additional info about the product because some developers might have seen these names the first time.

e.g.
```md
- [Joy UI](…): Our new in-house library with fresh design.
- [Base UI with Tailwind CSS]: A collection of unstyled components.
```

cc @samuelsycamore

Reply:
I've updated it to:
```
Material UI - A React UI library that implements Material Design
Joy UI - Our newest customizable UI library designed to spark joy
Base UI – A collection of headless components and hooks
```